mypy.ini

```diff
@@ -0,0 +1,2 @@
+[mypy-dpath.*]
+ignore_missing_imports = True
```

Comment:
@gruebel What's this?

Reply:
I use `mypy` for type checking. When more and more of the project has type hints added, then I would also add a github action to check the typing with `mypy` for each PR. And this tells `mypy` to ignore the `dpath` import, because that library doesn't have any type hints included.